ios/NativeModules/TextileNode.m

```diff
@@ -187,11 +199,8 @@ - (dispatch_queue_t)methodQueue {
 RCT_EXPORT_METHOD(getPhotoBlocks:(NSString *)offset limit:(int)limit threadName:(NSString *)thread resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
   NSError *error;
   NSString *jsonString = [self _getPhotoBlocks:offset withLimit:limit fromThreadNamed:thread error:&error];
-  NSData *data = [jsonString dataUsingEncoding:NSUTF8StringEncoding];
```

Comment:
Removed the JSON serialization from here, will do it in JS layer. TODO: Remove JSON stuff from Android impl, just resolve with the String